website/docs/r/elasticache_user.html.markdown

```diff
@@ -25,6 +25,19 @@ resource "aws_elasticache_user" "test" {
 }
 ```
 
+```terraform
+resource "aws_elasticache_user" "test" {
+  user_id       = "testUserId"
+  user_name     = "testUserName"
+  access_string = "on ~* +@all"
+  engine        = "REDIS"
+  
+  authentication_mode {
+    type = "iam"
```

Comment:
It would be good to show the slightly more complex example of `passwords` in the docs.

Reply:
added a password example. thanks for the suggestion